dev/drivers/scripts/wafs/stats/jevs_wafs_atmos_stats.sh

```diff
@@ -23,6 +23,14 @@ export COMPONENT=wafs
 export RUN=atmos
 export VERIF_CASE=grid2grid
 
+############################################################
+Set data directives
+############################################################
+export SENDECF=${SENDECF:-YES}
+export SENDCOM=${SENDCOM:-YES}
+export SENDDBN=${SENDDBN:-NO}
+export SENDMAIL=${SENDMAIL:-NO}
+
```

Comment:
@YaliMao-NOAA These four lines need to be in the J-job, not the driver script.  Place these lines in JEFS_WAFS_ATMOS_STATS and JEVS_WAFS_ATMOS_PLOTS, and remove them from here.

Here's an example, from the current branch I am working in:

https://github.com/PerryShafran-NOAA/EVS/blob/bugfix/analyses_EE2/jobs/JEVS_ANALYSES_STATS

Thanks!

Perry